innovation_sweet_spots/utils/altair_save_utils.py

```diff
@@ -4,21 +4,20 @@
 Functions to save altair charts
 """
 from altair_saver import save
-from selenium import webdriver
-from webdriver_manager.chrome import ChromeDriverManager
+import chromedriver_autoinstaller
 import os
 from typing import Iterator
 from innovation_sweet_spots import PROJECT_DIR
-import logging
+from selenium import webdriver
 
 FIGURE_PATH = f"{PROJECT_DIR}/outputs/figures"
 DEFAULT_FILETYPES = ["png", "svg", "html"]
 
 
 def google_chrome_driver_setup():
-    """Set up the driver to save figures"""
-    driver = webdriver.Chrome(ChromeDriverManager().install())
-    return driver
+    """Set up and return the driver to save figures"""
+    chromedriver_autoinstaller.install()
+    return webdriver.Chrome()
```

Comment:
I had issues installing chromedriver. Using chromedriver_autoinstaller worked for me.